### ISSUE TITLE
Hat Input On Official Wireless 360 Controller Different Events

### ISSUE THREAD

**Issue by jennib3** [linusg/xbox360controller#9]
xbox360controller version 1.1.2
ARM64 Ubuntu 16.04
xpad version 0.4 (https://github.com/paroj/xpad)

Input events for the hat axis are listed as events 6,7. These events only post when the hat is moved out of that position (i.e. hat moved from 0 ->-1 posts event for value 0, then a moved from -1 -> 0 posts event for -1) or when another event is posted (i.e. no event posted when moving from 0 -> 1 on hat, but once a trigger is pulled the event for hat position 1 posts).

On this system the events that corresponded with hat input were as follows:

```
        if event.number == 11:
            self.hat._value_x = -int(event.value)
        if event.number == 12:
            self.hat._value_x = int(event.value)
        if event.number == 13:
            self.hat._value_y = int(event.value)
        if event.number == 14:
            self.hat._value_y = -int(event.value)

**Comment by linusg** [comment]
Thanks for reporting! I don't have the wireless controller so I can't test this myself, but are you by any chance able to provide a PR?
Thanks!

**Comment by jennib3** [comment]
I can provide a PR, but I'm only able to test with the wireless 360 controller and the xbox one bluetooth controller.

**Comment by linusg** [comment]
That’d be awesome. No worries, I can do the testing with the regular 360 wired controller.

**Comment by linusg** [comment]
Should be fixed by #18.
